python/paddle/v2/framework/tests/test_tensor.py

```diff
@@ -11,10 +11,10 @@ def test_int_tensor(self):
 
         tensor = var.get_tensor()
 
-        tensor.set_dims([1000, 784])
+        tensor.set_dims([100, 84])
```

Comment:
Can you try three configurations -- [7, 0], [0, 7], [7,13]  -- 7 and 13 are just small prime numbers.  Prime numbers are reasonable becaue they are not power of 2 or any way defactorizable. 0 is a good boundary case.